hadoop-hdds/docs/content/shell/KeyCommands.md

```diff
@@ -155,3 +156,22 @@ The `key cat` command displays the contents of a specific Ozone key to standard
 ozone sh key cat /hive/jan/hello.txt
 {{< /highlight >}}
 Displays the contents of the key hello.txt from the _/hive/jan_ bucket to standard output.
+
+### Cp
+
+The `key cp` command copies a key to another one in the specified bucket.
+
+***Params:***
+
+| Arguments                      |  Comment                                |
+|--------------------------------|-----------------------------------------|
+|  Uri                           | The name of the bucket in **/volume/bucket** format.
+|  FromKey                       | The existing key to be copied
+|  ToKey                         | The name of the new key
+| -r, \-\-replication            | Optional, Number of copies, ONE or THREE are the options. Picks up the default from cluster configuration.
+| -t, \-\-type                   | Optional, replication type of the new key. RATIS and STAND_ALONE are the options. Picks up the default from cluster configuration.
+
+{{< highlight bash >}}
+ozone sh key cp /hive/jan sales.orc new_one.orc
```

Comment:
The parameter is a bit different from hadoop cp comamnd. My understanding is that the uri is to help the OM HA but seems here we are restricted to under the same URI. Does it take full uri as from and to key parameters?

Reply:
Based on the code, the answer is no. You can use a bucketAddress (url or vol/bucket) + fromKey (string) + toKey (string).

This patch looks good to me as it describe the current behavior.

@xiaoyuyao Do you have any suggestion to change the code before the next release for better usability (btw. `ozone sh key put DESTINATIONURL localfile` is also reported to be confusing.)

Reply:
@elek yes. Let's file a ticket to fix it separately.